doc/howto/network_bridge_resolved.md

```diff
@@ -70,6 +70,59 @@ You must repeat the commands after each reboot and after LXD is restarted, or ma
 
 ## Make the `resolved` configuration persistent
 
+### Solution "A"
+
+You can automate the `systemd-resolved` DNS configuration, so that it is applied on system start and takes effect when LXD creates the network interface.
+
+To do so, create a `systemd` network file named `/etc/systemd/network/<network_bridge>.network` with the following content:
+
+```
+[Match]
+Name=<network_bridge>
+[Network]
+DNS=<dns_address>
+Domains=~<dns_domain>
+```
+
+So for example, `/etc/systemd/network/lxdbr0.network` with the following content (your IP, of course):
+
+```
+[Match]
+Name=lxdbr0
+[Network]
+DNS=10.167.146.1
+Domains=~lxd
+```
+
+If you have NOT freshly installed `lxd` (you've rebooted since install), then you only need to reload `systemd-resolved`:
+
+    sudo systemctl restart systemd-resolved.service
+
+In case you **haven't rebooted yet since first initializing `lxd`**, then you need to restart `lxd` first or just reboot (see https://github.com/canonical/lxd/issues/14588):
+
+    snap restart lxd
+
+You can test that the configuration got applied by running:
+
+    sudo resolvectl status
+
+The output should contain a part similar to this (you can see the DNS server and the `.lxd` domain mentioned):
+
+```
+[...]
+Link 4 (lxdbr0)
+    Current Scopes: DNS
+         Protocols: -DefaultRoute +LLMNR -mDNS -DNSOverTLS DNSSEC=no/unsupported
+Current DNS Server: 10.167.146.1
+       DNS Servers: 10.167.146.1
+        DNS Domain: ~lxd
+[...]
+```
+
+This solution "A" is better than solution "B" in the sense that it is more resilient, and gets reapplied if the `lxdbr0` network is re-created for some reason (LXD update+restart occasionally apparently does this).
+
```

Comment:
If that solution "A" works reliably (and doesn't require a reboot), I think we'd outright want to get rid of solution "B".

Reply:
It's up to you, but I personally would leave it there for a while. I'm more reluctant to remove documentation than adding it. :) Maybe remove it after a year or so? But as you say.